src/Telezhnaia.tex

```diff
@@ -71,10 +71,10 @@ \section{\uppercase{Working Experience} (3.5+ years total)}
 \end{spacing}
 \vspace{0.2in}
    \begin{itemize} \itemsep -2pt  % reduce space between items
-    \item \textbf{SWE at JetBrains}, Educational Products (Kotlin, Java, git)~--- Supported Go language in Educational plugin, improved intergation with \href{https://stepik.org/}{Stepik} and \href{https://hyperskill.org}{JetBrains Academy}. (Aug 2019 – now)
-   \item {\textbf{Senior SWE at Security Code}} (Python, Flask, RabbitMQ, PostgreSQL, SQLAlchemy, Marshmallow)~--- Developed backend for web client of security tool. Sped up transport component between core and Python backend up to 200\% by replacing self-written component to GRPC. Improved data validation component by using OpenAPI. (Sep 2018 – Aug 2019)
+    \item \textbf{SWE at JetBrains}, Educational Products (Kotlin, Java, git)~--- Supported Go language in Educational plugin, improved integration with \href{https://stepik.org/}{Stepik} and \href{https://hyperskill.org}{JetBrains Academy}. (Aug 2019 – now)
+   \item {\textbf{Senior SWE at Security Code}} (Python, Flask, RabbitMQ, PostgreSQL, SQLAlchemy, Marshmallow)~--- Developed backend for web client of security tool. Sped up transport component between core and Python backend by up to 200\% by replacing a self-written component with GRPC. Improved data validation component by using OpenAPI. (Sep 2018 – Aug 2019)
    \item {\textbf{Technology Evangelist at Microsoft}} (C\#, ML, Linux, Azure)~--- Developed internal projects, shared expertise at conferences, wrote technical articles. (Feb 2017 – Sep 2018)
-   \item {\textbf{SWE intern at Yandex}} (C++, Python, MapReduce, Bash, svn, regex)~--- Improved quality of search by geographical objects with help of linguistic features of Russian language. Supported sides of the world in queries. (Summer 2016)
```

Comment:
I'm fairly certain that you meant things like "north", "south", etc. and if so, in English they're called cardinal directions. Also "supported" means "was supporting existing functionality for a while" and I think you meant "added new functionality", so I changed it accordingly.